document/core/appendix/custom.rst

```diff
@@ -68,7 +68,7 @@ Name Maps
 .........
 
 A *name map* assigns :ref:`names <syntax-name>` to :ref:`indices <syntax-index>` in a given :ref:`index space <syntax-index>`.
-It consists of a :ref:`vector <binary-vec>` of index/name pairs in arbitrary order.
+It consists of a :ref:`vector <binary-vec>` of index/name pairs in order of increasing index value.
```

Comment:
That's not the definition I'm used to. Mine was more in line with:
http://mathworld.wolfram.com/topics/IncreasingandDecreasing.html
Even if monotonic is implied, I'd rather have the spec be (slightly) redundantly correct than minimally and ambiguously correct.